appdata.patch

```diff
@@ -0,0 +1,18 @@
+--- a/iconexplorer/CMakeLists.txt
++++ b/iconexplorer/CMakeLists.txt
+@@ -4,4 +4,4 @@ if(BUILD_TESTING)
+ endif()
+ 
+ install(FILES org.kde.iconexplorer.desktop DESTINATION ${KDE_INSTALL_APPDIR})
+-install(FILES org.kde.iconexplorer.appdata.xml DESTINATION ${KDE_INSTALL_METAINFODIR})
++install(FILES org.kde.plasma.iconexplorer.appdata.xml DESTINATION ${KDE_INSTALL_METAINFODIR})
+--- a/iconexplorer/org.kde.iconexplorer.appdata.xml
++++ b/iconexplorer/org.kde.plasma.iconexplorer.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop">
+-  <id>org.kde.iconexplorer</id>
++  <id>org.kde.plasma.iconexplorer</id>
```

Comment:
you have to match the id in manifest too, plus desktop filename and icons, not sure if those are done.

Reply:
Manifest and icon is fixed, desktop file already uses the correct ID.